src/xia2/Modules/Report/__init__.py

```diff
@@ -74,6 +78,16 @@ def __init__(
     ):
 
         self.params = params
+        self.n_bins = PhilIndex.params.xia2.settings.merging_statistics.n_bins
```

Comment:
Two questions, perhaps for @graeme-winter:
1. Am I using `xia2.Handlers.Phil.PhilIndex` correctly, or should I be using its `get_python_object` method?
2. Until I introduced this line, `Report`'s parameters could all be passed in as an attribute at initialisation time, albeit as a cumbersom `phil.scope_extract` object.  Now I'm getting `PhilIndex` and I don't fully understand when its values get set.  Am I introducing any subtle weirdness?

Reply:
1. use of `PhilIndex` looks fine
2. I don't know if you are adding subtle weirdness but your concern is valid - however I think if you have a `PhilIndex` then I think this is functionally the same as a `scope_extract`?

Reply:
OK, thanks.  I try to remain ignorant of the weird workings of PHIL, so if you don't have any concerns, nor do I.  It's true that `PhilIndex`'s type is a `scope_extract`, so it's more a question of when the scope is extracted (xia2 has lots of import-time initialisation of stuf like this) and whether that matters.

I think if you don;t have any concerns, we can probably merge it and find out the hard way.